scipy/special/_support_alternative_backends.py

```diff
@@ -19,6 +20,12 @@ def get_array_special_func(f_name, xp, n_array_args):
     if is_numpy(xp):
         f = getattr(_ufuncs, f_name, None)
     elif is_torch(xp):
+        # These functions are not implemented.
+        if f_name in {'gamma', 'betaln', 'beta', 'betainc', 'expi', 'expn'}:
+            raise NotImplementedError
+        # These functions have incompatible interaces.
+        if f_name in {'logsumex'}
```

Comment:
```suggestion
        if f_name in {'logsumexp'}
```
But this can just be removed. `scipy.special.logsumexp` has a generic implementation, and it does not go through this machinery.

Reply:
Oh, so it will call through the other Array API functions?

Reply:
It is implemented in terms of array API standard calls.
https://github.com/scipy/scipy/blob/25ca572f6dcaf2d7ad38d96fdc9def33b2f8a385/scipy/special/_logsumexp.py#L13

Reply:
Awesome!